deploy/vagrant/provision/roles/tachyon/files/set_journal_folder.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# For Tachyon version that is no less than 0.8
+echo 'tachyon.master.journal.folder=${tachyon.underfs.address}/tachyon/journal/' >> /tachyon/conf/tachyon-site.properties
+
+# For earlier Tachyon versions
+sed -i "s/tachyon.master.journal.folder=.*/tachyon.master.journal.folder=\$TACHYON_UNDERFS_ADDRESS\/tachyon\/journal\//g" /tachyon/conf/tachyon-env.sh
```

Comment:
Is there any way to avoid doing both attempts? What causes site properties to not be read in versions less than 0.8?